src/easeljs/display/Container.js

```diff
@@ -180,7 +180,7 @@ this.createjs = this.createjs||{};
 	 * 		container.addChild(bitmapInstance, shapeInstance, textInstance);
 	 *
 	 * @method addChild
-	 * @param {DisplayObject} child The display object to add.
+	 * @param {DisplayObject|...DisplayObject} child The display object to add.

```

Comment:
Looks like it should be 
`@param {DisplayObject} child* The display object to add.`
or if you update the grunt-contrib-yuidoc package then the new syntax is
`@param {DisplayObject} ...child The display object to add.`